src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery(new ColorSupplier(new Random()), new Random());
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < 3; i++) {
```

Comment:
create a constant from 3 -> private static final int BALLS_NUMBER = 3, and use it